app/presenters/waste_carriers_engine/edit_form_presenter.rb

```diff
@@ -62,6 +62,8 @@ def registered_address
     end
 
     def registration_type
+      return if transient_registration.registration_type.blank?
```

Comment:
In the certificate, for lower tier, we always show `carrier, brokers or dealer` type. Shall we do the same here and default to that?

Reply:
Interesting, I didn't realise that about the certificate! I'm not sure, to be honest – it's not technically a value that has been set or can be modified, so I'm not sure there is much point in displaying it, especially if it's only for the eyes of back office users.

Reply:
Leave it off. The user never submitted the value and its not something that we would allow to be edited for a lower-tier registration.

So I believe its simpler, clearer and more 'correct' to just not show this at all for lower-tier.